src/main/java/com/aws/greengrass/authorization/AuthorizationHandler.java

```diff
@@ -72,17 +74,20 @@
 
     private final AuthorizationModule authModule;
     private final ReadWriteLock rwLock = new ReentrantReadWriteLock();
+    private final Map<String, String> variables = new ConcurrentHashMap<>();
```

Comment:
delete this, there shouldn't be any variable support added; only wildcards.

Reply:
Will we not be adding it, or because its already implemented in the configuration interpolation PR ?

Reply:
Yes, both.